packages/forms-web-app/src/sass/components/ui/file-upload.scss

```diff
@@ -0,0 +1,10 @@
+.document-file-upload {
+	&:focus {
+		+ .govuk-button {
```

Comment:
This could become `.document-file-upload__label`

Reply:
Apologies, I meant that the label element with the `.govuk-button` class could have another class added to the element which would be `.document-file-upload__label`. How the input was defined before was fine.